lib/omnitrade_client.dart

```diff
@@ -0,0 +1,24 @@
+library omnitrade_client;
+
+import 'package:chopper/chopper.dart';
+import 'src/services.dart';
+import 'package:http/http.dart' as http;
+
+class OmniCredentials {}
+
+/// A OmniClient.
```

Comment:
Do we need this comment? I mean, it's basically the name of the class.

Reply:
Those comments with `///` is for generator documentation, but I agree, I'll improve this comment